cypress.config.ts

```diff
@@ -33,7 +32,6 @@ export default defineConfig({
         ...manageUsersApi,
         ...tokenVerification,
 
-        ...frontendComponents,
```

Comment:
Have removed wiremock stubs for front-end components. If defaults to using fallbacks when run in Cypress.